design/gateway-apis-implementation.md

```diff
@@ -163,23 +169,26 @@ The output of this watcher is Envoy configuration.
 ## Detailed Design
 
 ### Configuration
-Contour will have an entry added to the config file for the Gateway it should watch.
+Contour will have an entry added to the config file for the `ControllerName` it should watch.
+The `ControllerName` takes the format of `projectcontour.io/<namespace>/contour` which requires the `<namespace>` param to be specified matching the namespace where Contour is deployed. 
```

Comment:
Since we're going to add Class logic to pick the oldest? The format of this probably doesn't matter, but should follow the upstream convention. 

Should also look to make sure the upstream admission controller also enforces the correct pattern.

Reply:
We can probably say something like "The `ControllerName` should take the format of`, since upstream _recommends_ that controllers namespace this, but does not _require_ it.